syntax = "proto3";
package flowable;

option go_package = "go.easyops.local/contracts/protorepo-models/easyops/model/flowable";





/*
部署
*/
message FlowableDeployment {
    /*
    ID
    */
    string id = 1 ;
    /*
    名称
    */
    string name = 2 ;
    /*
    部署时间(YYYY-MM-DDTHH:mm:sszzz)
    */
    string deploymentTime = 3 ;
    /*
    分类
    */
    string category = 4 ;
    /*
    父部署ID
    */
    string parentDeploymentId = 5 ;
    /*
    URL
    */
    string url = 6 ;
    /*
    tenant ID
    */
    string tenantId = 7 ;
}

